packages/client-lib/src/scheduling_client.ts

```diff
@@ -43,87 +43,94 @@ export class SchedulingClient {
 	// Properties received through the constructor.
 	private readonly logger: ILogger;
 	// Other properties.
-	private readonly httpClient: AxiosInstance;
+	private readonly URL_REMINDERS:string;
+    private defaultHeaders: Headers
 
 	constructor(
 		logger: ILogger,
 		URL_REMINDERS: string,
-		TIMEOUT_MS_HTTP_CLIENT: number
 	) {
 		this.logger = logger;
 
-		this.httpClient = axios.create({
-			baseURL: URL_REMINDERS,
-			timeout: TIMEOUT_MS_HTTP_CLIENT
-		});
+		this.URL_REMINDERS = URL_REMINDERS;
+        this.defaultHeaders = new Headers();
+        this.defaultHeaders.append("Content-Type","application/json")
 	}
 
 	async createReminder(reminder: IReminder): Promise<string> {
 		try {
-			const res = await this.httpClient.post("/", reminder);
-			return res.data.reminderId;
-		} catch (e: unknown) {
-			const serverErrorMessage: string | undefined = (e as AxiosError).message;
-			// istanbul ignore if
-			if (serverErrorMessage === undefined) {
-				this.logger.error(e);
-				throw new UnableToReachServerError(); // TODO.
+			const reqInit: RequestInit = {
+				method:"POST",
+                headers:this.defaultHeaders,
+				body:JSON.stringify(reminder)
 			}
+
+			const res = await fetch(`${this.URL_REMINDERS}/`,reqInit);
+			const data = await res.json();
+            // istanbul ignore if
+			if(!data.reminderId){
+				throw new Error(data.message);
+			}
+			return data.reminderId;
+		} catch (e: any) {
```

Comment:
Please run the lint on this, it should complain about the type being `any`. Check other BCs to see examples of how this is handled.